crates/storage-rocksdb/src/inbox_table/mod.rs

```diff
@@ -93,6 +95,51 @@ impl<'a> InboxTable for RocksDBTransaction<'a> {
             TableScanIterationDecision::Emit(inbox_entry)
         })
     }
+
+    /// Scans the inbox for an inbox entry with the given invocation id.
+    ///
+    /// Important: This method can be quite costly if it is invoked with an `InvocationId` because
+    /// it needs to scan all inboxes for the given partition key to match the given invocation uuid.
```

Comment:
I would put this comment on the iface definition.

Reply:
We _probably_ will need to do this also in CLI+observability operations. Maybe we should use a rocksdb index for this?

Reply:
Will move the docs to the interface definition.

The index is a good idea. I suggest to add it once we see that this operation becomes a bottleneck.